.eslintrc

```diff
@@ -130,6 +130,7 @@
     "oppia/dependency-checks": "error",
     "oppia/match-line-break": "error",
     "oppia/no-test-blockers": "error",
+    "oppia/disallow-angularjs-property": "error",
```

Comment:
I would suggest change `property to `properties` as there are two properties under this rule.

Reply:
Done!